recipes/bargeparse/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "bargeparse" %}
+{% set version = "0.8.2" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bargeparse-{{ version }}.tar.gz
+  sha256: 99c0dd179549e9a7381ff46f58ebda1488f0e355578abc419efc99cc52210e87
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  source_files:
```

Comment:
It is unclear to me why you need the `source_files` section. Typically this is for test data that's not included in the package. I don't see that here. If it is not needed, please remove it.

Reply:
Pytest could not find the tests when I built locally 🤷‍♂️.  I gather this is because pytest only collects from the working directory or supplied arguments.

I've tried `pytest $SRC_DIR` but no luck.  Looking in that directory showed only a bunch of conda specific files.  There is the `SP_DIR` var that points to site-packages but that seems a little brittle.

Manually including the tests is the only way I could get the tests to run. I could be a little more specific and have `- bargeparse/test_*.py`…

Thoughts?

Reply:
Ok, sounds reasonable. I think we should include the specific files as you suggested.

Reply:
Ok I've tried to selectively include the test files but I couldn't get the tests to work.  The problem is there's an import from conftest.py – including this as well doesn't fix the problem due to the relative import (an absolute import won't work here).

It's probably better off & simpler to just import the bargeparse/ directory.